integration/Main.purs

```diff
@@ -142,6 +143,12 @@ main = runAff_ (either throwException mempty) do
           ]
       Console.error message
 
+  let
+    mods = Array.mapMaybe _.mbModule moduleResults
+    sorted = sortModules mods
+
+  liftEffect $ Console.log $ "Sorted Module Graph for " <> show (Array.length sorted) <> " Modules"
```

Comment:
I added this to the integration test just to make sure that the `sortModules` function was returning the correct amount of modules. I was previously printing it out but it takes up a lot of space so I left that out.

Reply:
Should this assert in some way?

Reply:
We haven't put anything that asserts at all yet, but I added the same logic where we say how many modules we parsed out of the total amount of successfully parsed modules.